### PR TITLE
Publish updated cookie list for 2022/09/01

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by J3nnie — APPROVED** [review]

**Comment by kraniet888** [comment]
image

**Review by markoenix — COMMENTED** [review]
MarkoDeLaTorre

File: README.md [hunk 1]
```diff
@@ -2,16 +2,17 @@
 
 GitHub provides a great deal of transparency regarding how we use your data, how we collect your data, and with whom we share your data. To that end, we provide this page which details how we use cookies.
 
-GitHub uses cookies to provide and secure our websites, as well as to analyze the usage of our websites, in order to offer you a great user experience. Please take a look at our [Privacy Statement](https://docs.github.com/en/github/site-policy/github-privacy-statement#our-use-of-cookies-and-tracking) if you’d like more information about cookies, and on how and why we use them.
+GitHub uses cookies to provide and secure our websites, as well as to analyze the usage of our websites, in order to offer you a great user experience. Please take a look at our [Privacy Statement](https://docs.github.com/en/github/site-policy/github-privacy-statement#our-use-of-cookies-and-tracking-technologies) if you’d like more information about cookies, and on how and why we use them and cookie-related personal data. You can change your preference about non-essential cookies at any time by following the instructions in the `DNT` section of the Privacy Statement.
```
Comment (kraniet888) [inline README.md:5]: [😬]()

File: README.md [hunk 2]
```diff
@@ -2,16 +2,17 @@
...
 
-Since the number and names of cookies may change, the table below may be updated from time to time. Follow [these instructions](https://github.com/privacy/resources/blob/main/subscribe.md) for how to subscribe to changes in this repo
```
Comment (kraniet888) [inline README.md:7]: image

File: README.md (hunk 1 above)
Comment (kraniet888) [reply README.md:5]: Crack yo

File: README.md (hunk 2 above)
Comment (kraniet888) [reply README.md:7]: End

**Review by kraniet888 — COMMENTED** [review]
What?

**Review by kraniet888 — COMMENTED** [review]
Exit

**Review by kraniet888 — COMMENTED** [review]
Tm

**Pull request merged**